data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -91,18 +95,23 @@ def extract(self) -> None:
             constants.DATA_CENSUS_CSV_FILE_PATH
         )
 
-    def _transform_counties(self, initial_counties_df: pd.DataFrame) -> pd.DataFrame:
+    def _transform_counties(
+        self, initial_counties_df: pd.DataFrame
+    ) -> pd.DataFrame:
         """
         Necessary modifications to the counties dataframe
         """
         # Rename some of the columns to prepare for merge
         new_df = initial_counties_df[constants.CENSUS_COUNTIES_COLUMNS]
         new_df.rename(
-            columns={"USPS": "State Abbreviation", "NAME": "County Name"}, inplace=True
+            columns={"USPS": "State Abbreviation", "NAME": "County Name"},
+            inplace=True,
         )
         return new_df
 
-    def _transform_states(self, initial_states_df: pd.DataFrame) -> pd.DataFrame:
+    def _transform_states(
+        self, initial_states_df: pd.DataFrame
+    ) -> pd.DataFrame:
```

Comment:
This is [PEP8's official recommendation](https://realpython.com/python-pep8/#:~:text=PEP%208%20suggests%20lines%20should,to%20run%20over%20several%20lines.) and Black follows it.

Reply:
I think this is just `black` doing its thing! I don't have a strong opinion otherwise.

Reply:
Interesting! Okay, so following that link, I also see that while breaking up lines is recommended, this current style is not recommended. Instead the recommended style in the [Indentation section](https://realpython.com/python-pep8/#indentation-following-line-breaks) is to indent the continued function argument line even further:

```
def function(
        arg_one, arg_two,
        arg_three, arg_four):
    return arg_one
```

However, if Black is just doing this automatically, then it seems out of scope to address that in this PR. We can look into updating Black later to change the indentation during autocorrect if we want to.

Reply:
Yeah, since Black came out, these kind of conversations are just deferred to what Black does. You can get into a thousand opinions, and that was sort of the Python formatting world before Black came, so I suggest to follow whatever Black does.